doc/api/tls.md

```diff
@@ -1178,6 +1178,9 @@ being issued by trusted CA (`options.ca`).
 <!-- YAML
 added: v0.11.3
 changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/27816
+    description: The `hints` option is now supported.
```

Comment:
nit: "now" will soon be yesteryear, perhaps

> Support the "hints" option.

Reply:
It is to be consistent with the other changes. For example https://github.com/nodejs/node/commit/c6a2fdf3aa8f8db7344ae7c530468b9feb715086

"is now supported" is used in a lot for cases like this in the doc, how about fixing all of them at once in a separate commit?

Reply:
ok, I see it is common, so leave it as it is.